### PR TITLE
(#1579) Breadcrumb storybook reorganization

### LINKED ISSUE #1579: Update storybook structure for usa-breadcrumb
### `usa-breadcrumb` storybook structure
1. Default (folder)
    1. Default
2. Variants (folder)
    1. Wrap (folder)
        a. Wrap
3. Test cases (folder)
    1. Default with Very Long Content
    2. Wrap with Very Long Content

Other tasks:
* Clean up all of the regressions (redundant breakpoint scenarios) associated with this component
* Create ncids-twig template
* Make sure scenarios still reflect all of the test cases that we have, but we do not need them as examples in the storybook structure
* Patch print styles to breadcrumb. Remove `usa-breadcrumb` from internal

> **ESTIMATE** TBD

### Additional details / screenshot
- [Storybook structure wiki](https://github.com/NCIOCPL/ncids/wiki/Storybook-Structure)

### Related Tickets
* Parent ticket: #1224

### PR DESCRIPTION
**Pull request by arcepaul**
Closes #1579

- file renaming and re-organization

### CODE REVIEW

**Review by andyvanavery31 — CHANGES_REQUESTED** [review]
Could you please update the test cases to have the names "Default with Very Long Content" and "Wrap with Very Long Content" and update those two stories with the long breadcrumbs. Then could you also please update the reference images with those stories at the different breakpoints?

**Review by andyvanavery31 — CHANGES_REQUESTED** [review]
Can you please change the contents of the breadcrumb Test Cases of "Default with Very Long Content" and "Wrap with Very Long Content" to use the following breadcrumb: (as seen on [this page](https://www.cancer.gov/grants-training/grants-funding/funding-opportunities/oia/award-recipients/2022)

"Home > Grants & Training > Research Grants > Research Funding Opportunities > Outstanding Investigator Award > Award Recipients > 2022 NCI Outstanding Investigator Award Recipients". 

And then can you please make sure the regression images are updated with this change?

File: testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb--default.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<nav class="usa-breadcrumb" aria-label="Breadcrumbs,,">
+	<ol class="usa-breadcrumb__list">
+		<li class="usa-breadcrumb__list-item">
+			<a href="#" class="usa-breadcrumb__link">
+				<span>Home</span>
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb--default.twig:5]: Don't hardcode content - pass them through params

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

File: testing/ncids-css-testing/stories/components/usa-breadcrumb/content/usa-breadcrumb--extra-long-content.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		{
+			"label": "Research Funding Opportunities",
+			"href": "#"
+		},
+		{
+			"label": "Outstanding Investigator Award",
+			"href": "#"
+		},
+		{
+			"label": "Award Recipients",
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/components/usa-breadcrumb/content/usa-breadcrumb--extra-long-content.json:25]: https://github.com/NCIOCPL/ncids/pull/1621#pullrequestreview-2004298462

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

File: testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb.default.stories.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+export default {
+	title: 'Components/Breadcrumb/Default',
+	args: {},
+};
+
+const Template = (args) => Component(args);
+
+export const Default = () => (
+	<TestCase css={css} html={Template.bind({})(DefaultVeryLongContent)} />
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb.default.stories.js:17]: Well there's no point in the test case for very long content if everything has very long content.

File: testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb.scenarios.js [hunk 4]
```diff
@@ -1,10 +1,44 @@
...
+		label: 'usa-breadcrumb Wrap with Very Long Content',
+		storyId: 'components-breadcrumb-variants--wrap',
+		viewports: [
+			{
+				label: 'mobile',
+				width: 479,
+				height: 360,
+			},
+			{
+				label: 'tablet',
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb.scenarios.js:33]: There's only a breakpoint for mobile-lg in the scss. I'm not seeing a patch for $theme-breadcrumb-min-width to tablet.

1. Default.
2. Any breakpoints found in css or html classes
3. Larger screens if only they make sense for testing.

Replace tablet with mobile-lg.

File: testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb.scenarios.js [hunk 5]
```diff
@@ -1,10 +1,44 @@
...
+				width: 640,
+				height: 360,
+			},
+			{
+				label: 'desktop',
+				width: 880,
```
Comment (olitharp-nci) [inline testing/ncids-css-testing/stories/components/usa-breadcrumb/usa-breadcrumb.scenarios.js:39]: Desktop is not 880 it is 1024.

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Comment by andyvanavery31** [comment]
@arcepaul Default With Very Long Content in Test Cases is wrapping and it shouldn't. could you please that story and please update the regression images to have the correct images matching the names? We should have 

- Default at Mobile, Mobile Large, Tablet, Desktop
- Wrap at Mobile, Mobile Large, Tablet, Desktop
- Default Very Long at Mobile, Mobile Large, Tablet, Desktop
- Wrap Very Long at Mobile, Mobile Large, Tablet, Desktop

**Review by andyvanavery31 — APPROVED** [review]
Looks good to me!

**Review by andyvanavery31 — APPROVED** [review]
Looks good to me!

**Review by olitharp-nci — APPROVED** [review]

**Pull request merged**